test/jdk/javax/swing/JColorChooser/Test4319113.java

```diff
@@ -50,56 +38,65 @@
 import javax.swing.SwingUtilities;
 import javax.swing.UIManager;
 
-public class Test4319113
-extends JApplet
-implements ActionListener {
-    private final JFrame frame = new JFrame("frame");
-    private JComboBox cbPlaf;
-
-    @Override
-    public void init() {
-        try {
-            java.awt.EventQueue.invokeLater( () -> {
-                Test4319113.this.frame.setLayout(new GridLayout(2, 1));
-                Test4319113.this.show(Test4319113.this.frame);
-            });
-        }catch(Exception ex) {
-            ex.printStackTrace();
-        }
+/*
+ * @test
+ * @bug 4319113
+ * @library /test/jdk/java/awt/regtesthelpers /test/lib
+ * @build PassFailJFrame
+ * @summary Tests the open JColorChooser behavior on LaF change.
+ * @run main/manual Test4319113
+ */
+public class Test4319113 {
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame.builder()
+                .title("Test4319113")
+                .instructions("Choose a LaF using the dropdown. Click on the Show JColorChooser button.\n" +
```

Comment:
I feel its better to use a separate String variable for Test instructions and then use it here.